doc/nrf/software_maturity.rst

```diff
@@ -73,6 +77,14 @@ See the following table for more details:
      - Incomplete verification
      - Not applicable.
 
+API deprecation
+***************
+
+The **Deprecated** status is assigned to API that has gone through all maturity levels, but is being phased out.
+The deprecated API will be removed in one of future releases, no earlier than two releases after the deprecation is announced and only when the code has transitioned to not using the deprecated API.
```

Comment:
is this correct ?
I believe that all NCS code should have transitioned to the new API before we do any deprecation.

Are there any reasons why we want to allow ourselves to use a deprecated api ?

Reply:
What you state here should probably be the guideline when we are marking an API deprecated, but do we need to have that statement in external documentation. I think what we write here is okay as external documentation and will give us some flexibility internally (not having to transition to not using the deprecated API before adding the label).

Reply:
discussed offline.